contrib/pthreadw32/module.defs

```diff
@@ -1,7 +1,9 @@
 $(eval $(call import.MODULE.defs,PTHREADW32,pthreadw32))
 $(eval $(call import.CONTRIB.defs,PTHREADW32))
 
+# TODO: Unknown upstream url
```

Comment:
I normally use the pthreads lib supplied by my mingw installation.  I wonder if we can just eliminate this dependency?